docs/en/api/health-check.md

```diff
@@ -0,0 +1,9 @@
+# Check OAP healthiness
```

Comment:
I think this should be added here? 

https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-health-check/

Reply:
I added to the API catalog, as this is not about setup, it's API doc